packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
+   
+   Most installations of Nix are multi-user and require you to restart the nix daemon to make your `nix.conf` changes take effect. 
+   
+   You can check whether Nix is running in multi-user or single-user mode by running the following command:
+   
+   ```bash
+   ps aux | grep nix-daemon
+   ```
+   
+   If you see only the `grep` command, then you are using single-user mode and you don't have to do anything else. 
+   If you see a `nix-daemon` process, then you are using multi-user mode and you need to restart the process
+   
+   On macOS, run the following commands:
+   
+   ```bash
+   sudo launchctl stop org.nixos.nix-daemon
+   sudo launchctl start org.nixos.nix-daemon
+   ```
+   
+   On Linux, run the following command:
+   
+   ```bash
+   sudo systemctl restart nix-daemon.service
+   ```
+   
+3. Build `chainweb-node` by running a command similar to the following for a specific release `tag`, such as 2.24.1:
+
+   ```bash
+   nix build github:kadena-io/chainweb-node/<tag>
+   ```
+   
+   After starting the build, you should see messages similar to the following:
+   
+   ```text
+   copying path '/nix/store/8dyrf48cwvyqhvks5adxlk675qgm7pql-haskell-project-plan-to-nix-pkgs' from 'https://nixcache.chainweb.com'
+   ...
+   ```
+   
+   These messages indicate that the pre-built artifacts are being successfully downloaded from the cache. 
+   
+   When the build is finished, the directory with the `chainweb-node` source code contains a `result` subdirectory with a symbolic link to the Nix cache.
+
+4. Verify that `chainweb-node` is ready to use and review command-line configuration options by running the following command:
+   
+   ```bash
+   ./result/bin/chainweb-node --help
+   ```
+
+   You should see usage information about the configuration settings you can specify as command-line options similar to the following truncated output:
+
+   ```bash
+   Usage: chainweb-node [--info] [--long-info] [-v|--version] [--license] 
+                        [-?|-h|--help] 
+                        [--print-config-as full|minimal|diff | --print-config] 
+                        [--config-file FILE] 
+   ```
+
+   From the usage information, you can see that there are a large number of configuration options that you can use to control and operation and behavior of the Chainweb node. 
+   Before you start the node, you should review the configuration options and the default values to determine whether you want to make any changes to the configuration of the node.
+   For information about this next step, see [Review the default configuration](#review-the-default-configuration).
+
+## Review the default configuration
+
+Now that you have a Chainweb node binary ready to run, you should review the default configuration options and the default values. 
+Configuration options and settings control many types of node behavior, operation, and features.
+For example, you can use configuration options to enable or disable telemetry for the node.
```

Comment:
We could call it "user telemetry logging" or something here just to avoid the scariness of the word "telemetry"... "telemetry" usually sounds like "my data is being sent to someone else", which is not the case of chainweb-node telemetry logging, which is for operators.